python/constraints.txt

```diff
@@ -456,6 +456,8 @@ pyarrow==16.1.0
     # via
     #   -r python/requirements.in
     #   streamlit
+pyarrow-stubs==10.0.1.7
```

Comment:
perhaps I will answer my question later in the review... if not, curious where do we use this stub?

Reply:
When reading parquet metadata